Shared/mods/deathmatch/logic/lua/CLuaFunctionParser.h

```diff
@@ -33,6 +34,8 @@ class LuaFunctionError
     constexpr bool        IsWarning() const noexcept { return m_bWarning; }
 };
 
+using LuaVarArgs = std::optional<CLuaArguments>;
```

Comment:
Unused?

Reply:
It is used as an alias. Instead of doing `std:;optional<CLuaArguments>` you could do `LuaVarArgs`